conf/environments.js

```diff
@@ -102,6 +102,11 @@ module.exports = {
             ecmaVersion: 6
         }
     },
+    es7: {
```

Comment:
:+1: to @nzakas' point. Not sure if this is the best path forward and we can always add it if it turns out to be something good and wanted.

Concern: what happens when I set ES6 environment and then ecmaVersion to 7? Which one takes precedence? I'm guessing and hoping that ecmaVersion setting takes precedence but don't recall a doc specifically mentioning this.